### DISCUSSION TITLE
Quarkus cold-start performance on AWS Lambda

### DISCUSSION THREAD

**Discussion by maxday** [quarkusio/quarkus#31158] [Community]
Hey team!
I've recently launched a dashboard to benchmark all cold start durations across all runtimes and of course I've added Quarkus (cold start is about: 239ms)

<img width="690" alt="Screenshot 2023-02-14 at 11 58 03 AM" src="https://user-images.githubusercontent.com/864493/218805373-4c9a7525-b8f2-4da8-85fd-be6f0f02edcb.png">

If you want to see the full dashboard: https://maxday.github.io/lambda-perf
(fully open source)

**Comment by gsmet** [comment]
Hi!

I understand it is a JVM lambda? Have you tried to go native?

**Reply by maxday** [reply]
Hi!
it's native: https://github.com/maxday/lambda-perf/tree/main/s3-uploader/runtimes/quarkus_native_on_provided_al2
